### PR TITLE
Use base header on all screens && update sdk version && redirect to …

### PR DESCRIPTION
**Pull request by vryazanov**
…home if token is correct

### CODE REVIEW

File: src/components/Home.js [hunk 1]
```diff
@@ -1,28 +1,23 @@
...
-                    <Button title="Buy / Sell" />
-                </View>
-            </View>
-        )
-    }
+  render() {
+    return (
+      <View style={styles.header}>
+          <Button title="Transactions" />
+          <Button title="Pay" />
```
Comment (vladarefiev) [inline src/components/Home.js]: @vryazanov what is diff between pay and buy/sell?

File: src/ui/Header.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+import Logo from './Logo'
+import styles from '../styles'
+
+
+export default class Header extends PureComponent {
+  render() {
+    return (
+        <View style={styles.header}>
+            <Logo />
```
Comment (vladarefiev) [inline src/ui/Header.js]: @vryazanov too much spaces

File: src/components/Home.js (hunk 1 above)
Comment (vryazanov) [reply src/components/Home.js]: @vladarefiev buy/sell is about actions with Zold exchange platform, BTC -> ZLD / ZLD -> BTC.
Pay is about transfer between wallets inside Zold

File: src/ui/Header.js (hunk 2 above)
Comment (vryazanov) [reply src/ui/Header.js]: @vladarefiev oops, will fix

**Comment by vryazanov** [comment]
@vladarefiev fixed

**Review by vladarefiev — APPROVED** [review]

**Pull request merged**